src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .trim()
+    .split(';')
+    .map((pair) => pair.trim().split(':'))
+    .filter((pair) => pair.length > 1);
+
+  const styleObj = {};
+
+  stylePairs.forEach((pair) => {
```

Comment:
You are using the forEach method correctly here as it does not return a new array and is not modifying an existing one. Good job on that.